lib/config/default.js

```diff
@@ -1345,6 +1345,18 @@ defaultConfig.definition = () => ({
       formatter: boolean,
       default: false
     }
+  },
+
+  /**
+   * When enabled, large language model (LLM) events generated for
+   * supported instrumented LLM libraries.
+   */
+  llm_events: {
```

Comment:
I haven't had a chance to update the CDD but we're doing away with this. so it'll just be the feature flag.  Can you also document the feature flag in docs/feature_flags.md please?